BlazarUI/app/scripts/components/repo-build/InterProjectAlert.jsx

```diff
@@ -162,16 +178,16 @@ class InterProjectAlert extends Component {
       return null;
     }
 
-    const {upstreamRepoBuilds, downstreamRepoBuilds, interProjectBuildId} = upAndDownstreamModules;
+    const {upstreamRepoBuilds, downstreamRepoBuilds, rootRepoBuilds, interProjectBuildId} = upAndDownstreamModules;
 
-    if (Object.keys(upstreamRepoBuilds).length === 0 && Object.keys(downstreamRepoBuilds).length === 0) {
+    if (!(Object.keys(upstreamRepoBuilds).length + Object.keys(downstreamRepoBuilds).length  + Object.keys(rootRepoBuilds).length)) {
```

Comment:
I think http://underscorejs.org/#isEmpty represents a little more succinctly what you're trying to do.